core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -2933,6 +2940,112 @@ class KafkaController(val config: KafkaConfig,
       allocateProducerIdsRequest.brokerEpoch, eventManagerCallback))
   }
 
+  def processCorruptedBrokersChange(): Unit = {
+    if (!isActive) {
+      return
+    }
+    val corruptedBrokers = zkClient.getCorruptedBrokers.mapValues(_.clearedFromIsrs)
+    info(s"Received corrupted brokers change = $corruptedBrokers")
```

Comment:
Is this going to log every time one of them gets marked cleared? (Not a problem, just curious.)

Reply:
Yes, it would.